samples/charts/area/line-datasets.html

```diff
@@ -152,6 +152,18 @@
 			});
 			chart.update();
 		}
+
+		document.getElementById('togglePropagate').onclick = function() {
```

Comment:
Maybe better to use `addEventListener('click'`?

Reply:
Better to keep the onclick event handler inline, as DOM attribute (previous code)